.github/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 1. Enhancement - Added TDA D597 - thanks to @robbo599 (Lee Roberts)
 2. Bug - Swapped Stansted (EGSS) stands 64R and 64L around to correct positions - thanks to @rishab-alt
 3. Enhancement - Corrected Manchester (EGCC) Zulu lines to colours Orange and Blue - thanks to @rishab-alt (Rishab Saddiq)
+4. Airac - Jersey (EGJJ) changes to A3 and A4 position - thanks to @rishab-alt
```

Comment:
```suggestion
4. AIRAC (2305) - Changed Jersey (EGJJ) A3 and A4 positions - thanks to @rishab-alt
```
Per the standard format defined in the [style guide](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#changelog).

Reply:
i haven't been using the KMZ files included, just realized

Reply:
Again this has been marked as resolved without being changed, I'll do it myself.